app/models/game.rb

```diff
@@ -1,2 +1,6 @@
 class Game < ActiveRecord::Base
+
+  has_many :pieces
+  has_many :users
+  
```

Comment:
Associations should match with the column names.
(Rails knows to add the _id, which is why it is left out here)

We also have to tell it to use the User class.
```
belongs_to  :white_player, class_name: 'User'
belongs_to  :black_player, class_name: 'User'
belongs_to  :winner, class_name: 'User'
```